src/ImageGallery.jsx

```diff
@@ -965,6 +967,7 @@ export default class ImageGallery extends React.Component {
     const {
       infinite,
       preventDefaultTouchmoveEvent,
+      isRTL,
```

Comment:
Can you add a `defaultProps` of `isRTL` to false and also add it to `propTypes`

Reply:
Also, please update the `README.md` with the new prop and a description of what it does.